src/tests/base_test.py

```diff
@@ -82,7 +82,7 @@ def new_controller_configuration(self, request: FixtureRequest) -> BaseNodeConfi
         :rtype: new node controller configuration
         """
         if global_variables.platform == consts.Platforms.VSPHERE:
-            config = VSphereControllerConfig()
+            config = BaseVSphereConfig()
```

Comment:
not sure I understand
if it's now abstract, how can we instantiate it?

Reply:
Right, missed that. Fixed